apps/funke/authenticated-channel/requestPid.ts

```diff
@@ -0,0 +1,111 @@
+import { type Key, KeyType, getJwkFromKey } from '@credo-ts/core'
+import type { FullAppAgent } from '@package/agent'
+import { createPinDerivedEphKeyPop, deriveKeypairFromPin } from '../crypto/bPrime'
```

Comment:
we now have a src directory. Also how does this relate to receivePidBPrimeFlowUsecase ?